src/styles/blocks/main.scss

```diff
@@ -0,0 +1,20 @@
+.catalog {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+  justify-items: center;
+}
+
+@media (min-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(auto-fit, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(auto-fit, 200px);
+  }
+}
```

Comment:
The media query for min-width: 1024px is also redundant. The styles specified here are the same as the default styles for the .catalog class. You can safely remove this media query.